tests/test_generators.py

```diff
@@ -108,3 +152,24 @@ def test_semseg_shape_generator():
     assert im_shape == (BATCH_SIZE, IMAGE_SIZE, IMAGE_SIZE, 3)
     label_shape = item[1].shape
     assert label_shape == (BATCH_SIZE, IMAGE_SIZE, IMAGE_SIZE, len(config['classes']))
+
+
+def test_wrong_model_dataset_generator():
+    """Test a wrong model and wrong dataset
+    """
+    dataset = "fake"
+    model = "conquer_the_world"
+    IMAGE_SIZE = 10
+    BATCH_SIZE = 10
+    datapath = ("./tests/data/" + dataset + "/training")
+    config = {"classes": list('0123')}
+
+    # wrong dataset name
+    with pytest.raises(ValueError) as excinfo:
+        generator.create_generator(dataset, model, datapath, IMAGE_SIZE, BATCH_SIZE, config)
+    assert str(excinfo.value) == "Wrong dataset name fake"
+
+    # wrong model name
+    with pytest.raises(ValueError) as excinfo:
+        generator.create_generator('shapes', model, datapath, IMAGE_SIZE, BATCH_SIZE, config)
+    assert str(excinfo.value) == "Wrong model name conquer_the_world"
```

Comment:
```
assert str(excinfo.value) == "Wrong model name: {}".format(model)
```
(or any other valid string format in Python 3.5)

Reply:
done